.github/workflows/publish-runtime.yml

```diff
@@ -31,29 +31,45 @@ jobs:
 
   build-srtool-runtimes:
     needs: ["setup-scripts"]
-    runs-on: self-hosted
+    runs-on: bare-metal
     strategy:
       matrix:
         chain: ["moonbase", "moonriver", "moonbeam"]
         srtool_image:
           - purestake/srtool
         srtool_image_tag:
-          - 1.66.1
+          - 1.69.2
```

Comment:
If we build release runtime with rust 1.69, can you change at the same time the rust version in the `rust-toolchain` file?
I think it's better to keep the rust version synchronize between CI build and release build

Reply:
I think we should try to use the most recent stable version of srtool anyway, and use the same rust version in our `rust-toolchain` file. And if that doesn't work, then downgrade both at the same time.